### PR TITLE
Apple.com clone page

### PR DESCRIPTION
**Pull request by mariancristu**
# Completed project #4 - Apple.com clone

This project consisted of making a clone of a Apple.com page. I accomplished it by using:

- [x] Backgrounds
- [x] Gradients
- [x] Flex
- [x] Box model

## Project specific requirements:

- [x] Set up the top navigation bar.
- [x] Set up the main background image. 
- [x] Fill in and position the additional sections.

## Authors

👤 **Cristu Marian**
- Github: [@mariancristu](https://github.com/mariancristu)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        <a
+          href="https://web.archive.org/web/20140301004610/http://www.apple.com/iphone-5c/"
+        ></a>
+        <a
+          href="https://web.archive.org/web/20140301004610/http://www.apple.com/30-years/"
+        ></a>
+        <a
+          href="https://web.archive.org/web/20140301004610/http://www.apple.com/supplier-responsibility/"
+        ></a>
+      </div>
```
Comment (suretrust) [inline index.html]: These links should have images as shown on the original page.

**Yours**
images

**Original**
images2

Comment (mariancristu) [reply index.html]: I changed the relative links for the backgrounds and I changed the [Live Demo Link](https://raw.githack.com/mariancristu/Apple.com_clone_page/test/index.html) on Readme file. Thank you for your review!

**Comment by Anna-Myzukina** [comment]
### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
### Hi Cristu! WOW Your project looks! :+1: 

### ✔️ Approved image

* Please remember that your branch name should be descriptive (e.g., refactor-authentication, user-content-cache-key, make-retina-avatars), so that others can see what is being worked on. You can read more about it [here](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/). I hope in the next project you can do this!

screen


 #### You can merge this PR and move on with your projects. Don't forget to fill the project completion form.


*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*

**Comment by mariancristu** [comment]
Thank you for review and I will use a more descriptive branch name in the future.

**Pull request merged**